jasperreport/6.1.1.wso2v1/pom.xml

```diff
@@ -0,0 +1,99 @@
+                            org.jaxen.*; version="[1.1.0,2.0.0)",
+                            org.eclipse.jdt.*; version="[0.0.0,1.0.0)",
+                            org.codehaus.groovy.*; version="[2.3.0,3.0.0)",
+                            org.apache.xpath.*; version="[0.0.0,1.0.0)",
+                            org.apache.velocity.*; version="[2.4.0.wso2v1,3.0.0)",
+                            org.apache.tools.ant.*; version="[0.0.0,1.0.0)",
+                            org.apache.poi.*; version="[0.0.0,1.0.0)",
+                            org.apache.lucene.*; version="[5.2.0.wso2v1,6.0.0)",
+                            org.apache.http.util; version="[4.3.0.wso2v1,5.0.0)",
+                            org.apache.http.message; version="[4.3.0.wso2v1,5.0.0)",
+                            org.apache.http.impl.*; version="[4.3.0.wso2v2,5.0.0)",
+                            org.apache.http.conn; version="[4.3.0.wso2v2,5.0.0)",
+                            org.apache.http.client.*; version="[4.3.0.wso2v2,5.0.0)",
+                            org.apache.http.auth; version="[4.3.0.wso2v2,5.0.0)",
+                            org.apache.http; version="[4.3.0.wso2v1,5.0.0)",
+                            org.apache.commons.logging; version="[1.2.0,2.0.0)",
+                            org.apache.commons.lang; version="[2.6.0,3.0.0)",
+                            org.apache.commons.collections.*; version="[3.2.0,4.0.0)",
+                            jxl.*; version="[0.0.0,1.0.0)",
+                            javax.xml.xpath; version="[0.0.0,1.0.0)",
+                            javax.xml.transform.*; version="[0.0.0,1.0.0)",
+                            javax.xml.soap; version="[1.0.0,2.0.0)",
+                            javax.xml.parsers; version="[0.0.0,1.0.0)",
+                            javax.xml.namespace; version="[0.0.0,1.0.0)",
+                            javax.swing.*; version="[0.0.0,1.0.0)",
+                            javax.sql.*; version="[0.0.0,1.0.0)",
+                            javax.servlet.*; version="[2.6.0,3.0.0)",
+                            javax.print.*; version="[0.0.0,1.0.0)",
+                            javax.persistence; version="[2.0.0,3.0.0)",
+                            javax.naming; version="[0.0.0,1.0.0)",
+                            javax.imageio; version="[0.0.0,1.0.0)",
+                            groovyjarjarasm.asm; version="[2.3.0,3.0.0)",
+                            groovy.lang; version="[2.3.0,3.0.0)",
+                            com.lowagie.text.*; version="[2.1.0,3.0.0)"
+                        </Import-Package>
+                        <Include-Resource>
+                            @jasperreports-6.1.1.jar!/default.jasperreports.properties,
+                            @jasperreports-6.1.1.jar!/jasperreports_extension.properties
+                        </Include-Resource>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
Even i imported all the packages it gave an unresolved reference compilation error without dynamic import.